frontend/sentiment-dashboard/src/app/dashboard/dashboard.component.html

```diff
@@ -5,7 +5,7 @@ <h1 class="sr-only">Home</h1>
 </div>
 
 <div [hidden]="!currentlyComparing">
-  <app-form #form2 [showCompare]="false"></app-form>
+  <app-form #form2 [showCompare]="false" (filterChange)="updateStatsSubscription2($event)"></app-form>
```

Comment:
Can we use a more descriptive name for this? Maybe "updateCompareStatsSubscription"?